tests/data/spe1_st/tests/test_webviz_ert.py

```diff
@@ -37,7 +37,7 @@ def test_webviz_parameter_comparison(get_ensemble_id, dash_duo):
 
     ensemble_name = select_first(dash_duo, "#" + plugin.uuid("ensemble-multi-selector"))
     _verify_keys_in_menu(
-        dash_duo, plugin, parameter_keys, "parameter-selector-multi-params"
+        dash_duo, plugin, parameter_keys, "parameter-selector-multi-param"
```

Comment:
why the renaming from params -> param?

Reply:
because i think a jenkins test fails otherwise

so this is a bit of a problem - because I'm not sure if I was just to eager, or if I did good, but I remember seeing an inconsistency with respect to using `param` vs. `params` - and in particular where the string was given as `suffix`, it was `param`, so I believe I changed it to `param` in some places. but i didn't see the extent of it (there are many places where it's still params).

so i've encountered the exact same problem in another PR and "fixed" it there in the same spot, too... perhaps this needs a proper clean up, in a separate issue

Reply:
oh yeah and I don't understand why that jenkins test suddenly started failing

the change seems to be due to 3bc13f629ea0f890bf7796679aacdef539dffafb though, where the suffix "params" was replaces with `DataType.PARAMETERS` which has `value`: `param`

but how that change got merged, and now jenkins is failing, is a riddle to me

maybe it didn't fail on this PR either, and I'm just making up that memory - maybe i just happened to grep for params and thought that I missed it, or something!? I don't know, I'm sorry :disappointed:

Reply:
hmm looking at the changes I see that I changed it in many many spots, and not just in the line you're pointing out :sweat_smile: 

In that case, you can partly disregard the bla bla up there, and my answer is actually: to make things more consistent. maybe this shouldn't have been part of this PR though.

If you wish, I can revert the renames (where that's possible without breaking things after a rebase), just let me know.

Reply:
yupp after rebase all these renames are gone! gonna resolve this : )